### ISSUE TITLE
Noto Serif font minus sign misaligned

### ISSUE THREAD

**Issue by Arnache** [google/fonts#8765]
The character U+2212 (minus sign) shoud be vertically aligned at the level of the horizontal bar of the plus sign + (and half way of the equal sign =). This is the case in the current Noto Sans font but not the case in the Noto Serif font.

**Comment by thlinard** [comment]
Other issues (×<>+÷−¬=≠±):

1. The minus issue is also found in Noto Sans Italic.
2. For Noto Sans and Noto Serif, the + should be made of 2 bars of identical length. Once you have the correct dimension of the horizontal bar, you add two dots to it and you get the ÷, you duplicate it and you get the =, etc.
3. The × should be a +, with the only difference being a 45° inclination (correct for Noto Sans, if its size is the same as that of the +).
4. For Noto Serif (Roman and Italic), and Noto Sans Italic, there is a issue with ≠.
5. Alignment could indeed be improved (× is too high for Noto Serif, as noticed by the creator of this issue, < and > are too high for Noto Sans).

<img width="463" alt="Image" src="https://github.com/user-attachments/assets/9fa9db96-c312-4993-b592-4214c94ef933" />

**Comment by Arnache** [comment]
- It seems that Noto Serif has no "minus sign" glyph, though it has × + ÷ , which seems a bit surprising.
- What I and thlinard got for the minus sign probably are replacement glyphs from another font by our systems.

**Comment by thlinard** [comment]
> * It seems that Noto Serif has no "minus sign" glyph, though it has × + ÷ , which seems a bit surprising.
> 
>     * What I and thlinard got for the minus sign probably are replacement glyphs from another font by our systems.

No, the fonts have a minus glyph (coded U+2212), but it is only made of one component: the hyphen glyph.

**Comment by Arnache** [comment]
> No, the fonts have a minus glyph (coded U+2212), but it is only made of one component: the hyphen glyph.

Ah, thanks. How can I check this kind of things?

**Comment by thlinard** [comment]
You can use a font editor. Btw, the demo version (unlimited in time) of FontLab is sufficient to do that.

**Comment by Arnache** [comment]
> You can use a font editor. Btw, the demo version (unlimited in time) of FontLab is sufficient to do that.

Thanks. I just opened my (up to date) Ubuntu's NotoSerif-Regular.ttf and the Glyph for U+2212 MINUS SIGN is absent.

**Comment by simoncozens** [comment]
>  (up to date) Ubuntu's NotoSerif-Regular.ttf with FontForge and the Glyph for U+2212 MINUS SIGN is absent.

In which case, Ubuntu's Noto Serif is not up to date. :-)

**Comment by Arnache** [comment]
> > (up to date) Ubuntu's NotoSerif-Regular.ttf with FontForge and the Glyph for U+2212 MINUS SIGN is absent.
> 
> In which case, Ubuntu's Noto Serif is not up to date. :-)

I just checked: the system's font file is indeed dated dec 26th 2020 while Ubuntu is Ubuntu 24.04.1 LTS

**Comment by emmamarichal** [comment]
Hi @simoncozens! 

I can reproduce all of this (Chrome / Mac 12.0.1)

<img width="440" alt="Image" src="https://github.com/user-attachments/assets/4c387674-0e63-4543-ad18-70fd3566b136" />
